codalab/apps/web/static/js/worksheet/table_bundle_interface.js

```diff
@@ -32,11 +32,28 @@ var TableBundle = React.createClass({
             else
               this.focusOnRow(newIndex);
         }.bind(this), 'keydown');
+
+        Mousetrap.bind(['enter'], function(e) {
+            window.open(this.refs['row' + this.state.rowFocusIndex].props.url, '_blank');
+        }.bind(this), 'keydown');
+
+        // Paste uuid of focused bundle into console
+        Mousetrap.bind(['u'], function(e) {
+            var uuid = this.refs['row' + this.state.rowFocusIndex].props.uuid;
+            $('#command_line').terminal().insert(uuid);
+        }.bind(this), 'keydown');
+
+        // Paste args of focused bundle into console
+        Mousetrap.bind(['a'], function(e) {
+            var bundleInfo = this.refs['row' + this.state.rowFocusIndex].props.bundleInfo;
+            if (bundleInfo.command != null) {
+                $('#command_line').terminal().insert(bundleInfo.command);
```

Comment:
This looks pretty hard to pull out:
https://github.com/kashizui/codalab-cli/blob/7af1516f63a149480996d5c3e93fee541ab6cc9c/codalab/lib/worksheet_util.py#L354-L354

What do you think would be the best strategy here?